packages/babel-traverse/src/scope/binding.js

```diff
@@ -82,6 +82,9 @@ export default class Binding {
    */
 
   reference(path: NodePath) {
+    if (this.referencePaths.indexOf(path) !== -1) {
```

Comment:
This is definitely preference, maybe see if there are other places in the codebase so we can be consistent? I always do `!== -1`.